openapi/components/schemas/StorefrontQuoteChangeOrder.yaml

```
@@ -227,6 +227,38 @@ properties:
     description: |-
       Specifies if the quotation invoice must include interim items only.
       If this value is set to `false`, all upcoming items are included.
+  usageSettings:
+    type:
+      - 'object'
+      - 'null'
+    description: Settings for items with metered billing.
+    properties:
+      policy:
+        type: string
+        enum:
+          - reset
+          - transfer
+        description: Policy for removed items with metered billing plan.
```

suggestion: for readability
```suggestion
        description: Policy for removed items that have a metered billing plan.
```